api/controllers/blobs/create.js

```diff
@@ -32,17 +30,11 @@ module.exports = {
   fn: async ({ source, plugin, configs }) => {
     sails.log("Creating blob");
 
-    const blobBase64 = crypto
-      .createHash("sha256")
-      .update(Snippet.salt)
-      .update(JSON.stringify({ source, plugin, configs }))
-      .digest("hex");
-
     // Check for Blob and add if not present
-    const newBlob = await Blobs.findOrCreate(
-      { base64BlobKey: blobBase64 },
-      { base64BlobKey: blobBase64 }
-    );
+    const newBlob = await Blobs.create().fetch();
+    
+    // Add shareble link to response body
+    newBlob.url = `http://localhost:${sails.config.port}/share/${newBlob.id}`;
```

Comment:
Don't hardcode `localhost`, use an environment variable. See https://github.com/MLH-Fellowship/babel-sandbox/pull/62

Reply:
Actually, you can just give the directory. The client can assemble the url.

Reply:
Currently looking at this [link](https://sailsjs.com/documentation/reference/configuration/sails-config#?using-the-production-environment) to get ready for prod